dev_tools/install.sh

```diff
@@ -6,7 +6,7 @@ apt-get update
 apt-get -y install git;
```

Comment:
Is this file for installing software locally?

Reply:
Yes for Dev